app/presenters/queries/content_item_warnings.rb

```diff
@@ -0,0 +1,23 @@
+module Presenters
+  module Queries
+    module ContentItemWarnings
+      def self.call(content_id, state, base_path, document_type)
+        if state == "draft"
```

Comment:
perhaps a case for getting this condition out the way (and thereby reducing reducing the number of clauses one has to hold in memory whilst reading the code):

`return unless state == "draft"`